misc/extension_api_validation/4.2-stable.expected

```diff
@@ -14,3 +14,27 @@ Validate extension JSON: Error: Field 'classes/TileData/methods/get_navigation_p
 Validate extension JSON: Error: Field 'classes/TileData/methods/get_occluder/arguments': size changed value in new API, from 1 to 4.
 
 Added optional argument to get_navigation_polygon and get_occluder to specify a polygon transform.
+
+GH-84976
+--------
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/FinalAction/values/FINAL_ACTION_CONTINUE': value changed value in new API, from 2.0 to 0.
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/FinalAction/values/FINAL_ACTION_MAX': value changed value in new API, from 3.0 to 2.
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_CLEAR': value changed value in new API, from 0.0 to 1.
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_CLEAR_REGION_CONTINUE': value changed value in new API, from 2.0 to 0.
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_CONTINUE': value changed value in new API, from 5.0 to 0.
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_DROP': value changed value in new API, from 4.0 to 2.
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_KEEP': value changed value in new API, from 3.0 to 0.
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_MAX': value changed value in new API, from 6.0 to 3.
```

Comment:
I don't know that anyone is using these enums, so an argument could be made that the values changing is OK. However, if there's a way to maintain the original values, that would be ideal.

Reply:
The compatibility method does the mapping and under the deprecated flag there's basically aliasing of the old values to the new values. Is this the proper way to handle it?

https://github.com/DarioSamo/godot/blob/f23b0fdf62cdd0ffa27298531027978e29dcb4b9/servers/rendering/rendering_device.compat.inc#L45-L74

https://github.com/DarioSamo/godot/blob/f23b0fdf62cdd0ffa27298531027978e29dcb4b9/servers/rendering/rendering_device.h#L331-L353

Reply:
Ah, sure, if you've made compatibility methods for every method that accepts or returns this enum, then that's an acceptable solution. Usually, though, we just try to maintain enum values and add new ones at the end, because it can be a lot of work adding compatibility methods for everything that uses a given enum. If the number of methods is small enough that that can be done in this case, that's fine :-)

Reply:
It's only this one method that uses it (and the deprecated split one that was never used in the codebase because it just didn't work apparently). If you feel it'd be better to hack around the values a bit I'm fine with that too, but the scope is probably small enough that I'd probably prefer the cleaner version of the enum for the future.

Reply:
Ok, that seems fine!